python-bareos/bareos/bsock/lowlevel.py

```
@@ -60,12 +60,15 @@ def format_warning_short(message, category, filename, lineno, line=""):
     try:
-        import sslpsk
+        import sslpsk3
     except ImportError:
-        warnings.warn(
-            "Connection encryption via TLS-PSK is not available "
-            "(not available in 'ssl' and extra module 'sslpsk' is not installed)."
-        )
+        try:
+            import sslpsk
+        except ImportError:
+            warnings.warn(
```

Ruff (0.15.7)

[warning] No explicit `stacklevel` keyword argument found

Set `stacklevel=2`

(B028)